tools/make-source

```diff
@@ -26,26 +26,9 @@ fi
 base=$(cd $(dirname $0); pwd -P)
 outdir=$(pwd -P)
 
-# Make a .tar.gz with the source code
-
 cd $base/..
+./autogen.sh 1>&2
```

Comment:
Actually would suggest using ```mkdir -p dist``` or something like that. That is, if Makefile doesn't exist, create a dist directory and run autogen.sh in there. The reason for a different name in that case, is the lack of arguments passed to autogen.sh